appservice/src/deploy/runPreDeployTask.ts

```diff
@@ -65,18 +65,29 @@ function isTaskEqual(expectedName: string, expectedPath: string, actualTask: vsc
     // Example with prefix: "func: extensions install"
     const regexp: RegExp = new RegExp(`^(${actualTask.source}: )?${actualTask.name}$`, 'i');
     if (regexp.test(expectedName) && actualTask.scope !== undefined) {
-        const workspaceFolder: Partial<vscode.WorkspaceFolder> = <Partial<vscode.WorkspaceFolder>>actualTask.scope;
-        return !!workspaceFolder.uri && (isPathEqual(workspaceFolder.uri.fsPath, expectedPath) || isSubpath(workspaceFolder.uri.fsPath, expectedPath));
+        return isScopeEqual(actualTask, expectedPath);
     } else {
         return false;
     }
 }
 
-async function waitForPreDeployTask(preDeployTask: vscode.Task): Promise<number> {
+function isScopeEqual(task: vscode.Task, workspaceFsPath: string): boolean {
+    const workspaceFolder: Partial<vscode.WorkspaceFolder> = <Partial<vscode.WorkspaceFolder>>task.scope;
+    return !!workspaceFolder.uri && (isPathEqual(workspaceFolder.uri.fsPath, workspaceFsPath) || isSubpath(workspaceFolder.uri.fsPath, workspaceFsPath));
+}
+
+async function waitForPreDeployTask(preDeployTask: vscode.Task, deployFsPath: string): Promise<number> {
```

Comment:
Would like to see you fix the error message so that it says `Errors exist after running preDeployTask "clean"` instead of "publish-release"

Reply:
![image](https://user-images.githubusercontent.com/5290572/68255496-39dd3180-ffe2-11e9-992a-02a039862c61.png)